framework/test/mocha/integration/state_store/round_store.js

```diff
@@ -63,7 +63,7 @@ describe('system test - round store', () => {
 
 	describe('cache', () => {
 		it('should throw ', async () => {
-			expect(roundStore.cache()).to.eventually.throw(
+			return expect(roundStore.cache()).to.eventually.throw(
```

Comment:
Tests failing for this one. Changing to the following should fix it:

```suggestion
			return expect(roundStore.cache()).to.be.rejectedWith(
```

Reply:
@lsilvs cache is an async function right? So I think eventually should stay there.

Reply:
When you use `rejected` or `fulfilled` the `eventually` is implicit, I guess. But should work with it as well.